src/js/lib/content/src/index.js

```diff
@@ -438,6 +439,36 @@ function ok(value) {
     return value !== null && value !== undefined
 }
 
+/// Check whether the value is a string with value `"true"`/`"false"`, if so, return the
+// appropriate boolean instead. Otherwise, return the original value.
+function tryAsBoolean(value) {
```

Comment:
I think the name of the function could be more descriptive, like `parseBooleanOrLeaveAsIs` I know its longer, but "try" in name tells me that we are returning something like Option, but here we are returning the original value. It could also be `tryAsBooleanOrLeaveAsIs`, but `parse` is better as it tells that we are working with string input.